### PR TITLE
Add qtawesome to developer package lists

### PR DESCRIPTION
**Pull request by martyngigg**
Adds [`qtawesome`](https://pypi.python.org/pypi/QtAwesome) to developer package requirements. It is used for access the fontawesome icons and fonts.

**To test:**

* Code review

*No issue*

**Release Notes** 
*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec [hunk 1]
```diff
@@ -50,6 +50,7 @@ Requires: poco-devel >= 1.4.6
 Requires: PyQt4-devel
 Requires: python-qt5-devel
 Requires: python-QtPy
+Requires: python-qtawesome
```
Comment (peterfpeterson) [inline buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec]: Should be `python2-QtAwesome`

File: buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec [hunk 2]
```diff
@@ -111,6 +112,7 @@ Requires: python3-sip-devel
 Requires: python3-PyQt4-devel
 Requires: python-qt5-devel
 Requires: python3-QtPy
+Requires: python3-qtawesome
```
Comment (peterfpeterson) [inline buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec]: Correct the name and only use on fedora with ` %{?fedora:Requires: python3-QtAwesome}`

**Review by peterfpeterson — CHANGES_REQUESTED** [review]
I can only comment on the `.spec` changes

File: buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec (hunk 2 above)
Comment (martyngigg) [reply buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec]: Thanks. Just for my own learning why don't the python3 packages like `QtPy` need this restriction too?

File: buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec [hunk 3]
```diff
@@ -148,6 +150,9 @@ required for Mantid development.
 
 %changelog
 
+* Mon Jan 22 2017 Martyn Gigg <martyn.gigg@stfc.ac.uk>
```
Comment (peterfpeterson) [inline buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec]: Doesn't build because of wrong year. It is now 2018.

Comment (martyngigg) [reply buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec]: :man_facepalming:

**Review by peterfpeterson — APPROVED** [review]
`.spec` changes are ready.

**Pull request merged**